www/src/utils/createOgUrl.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+*/
+
+'use strict';
+
+import {urlRoot} from 'site-constants';
+
+export default markdownPath => {
+  // replace the file extention from .md to .html
+  const path = markdownPath.replace(/\.md$/, '.html');
+  // omit 'index.html' from og:url.
+  return `${urlRoot}/${/index\.html$/.test(path) ? '' : path}`;
```

Comment:
What's the advantage to trimming "index.html"?

This would clobber a path like "'some/blog/path/index.html'"

I suggest:
```js
path.replace(/index\.html$/, '')
```

Reply:
> What's the advantage to trimming "index.html"?

There might be not an advantage. But many links to `index.html` don't have `index.html` as the path. So I prefer to have consistency for these URLs.
But I'll remove the code if you don't need it.

> path.replace(/index\.html$/, '')

That makes sense. I've fixed it.